src/View/Parsers/ShortcodeParser.php

```diff
@@ -263,8 +263,8 @@ protected function insertListAfter($new, $after)
 		([^\s\/\'"=,]+)       # Name
 		\s* = \s*
 		(?:
-			(?:\'([^\']+)\') | # Value surrounded by \'
-			(?:"([^"]+)")    | # Value surrounded by "
+			(?:\'([^\']*)\') | # Value surrounded by \'
```

Comment:
The `+` character matches any sequence of at least one character matching `[^\']`. The `*` matches any sequence of character matching `[^\']` including no characters.

Because empty quotes wouldn't match the condition on line 266/267, they would end up matching the condition on line 268 which is meant to match attribute without quotes.